File: src/test/java/parse/LiveMicDemo.java
package parse;

import ai.picovoice.leopard.LeopardException;

import java.awt.*;
import java.io.IOException;

public class LiveMicDemo {
	
	public static void main(String[] args) throws IOException, InterruptedException, AWTException, LeopardException {
		LiveMic.startRecognition();
	}
	
}

File: src/test/java/parse/LiveMicDemo.kt
package parse

import ai.picovoice.leopard.LeopardException
import java.awt.AWTException
import java.io.IOException

object LiveMicDemo {
    @Throws(IOException::class, InterruptedException::class, AWTException::class, LeopardException::class)
    @JvmStatic
    fun main(args: Array<String>) {
        LiveMic.startRecognition()
    }
}
